extensions/php/package.nls.json

```diff
@@ -1,6 +1,7 @@
 ﻿{
-	"configuration.validate.enable": "Whether php validation is enabled or not.",
-	"configuration.validate.executablePath": "Points to the php executable.",
+	"configuration.builtInCompletions.enable": "Enable/disable built-in PHP completions.",
+	"configuration.validate.enable": "Enable/disable built-in PHP validation.",
+	"configuration.validate.executablePath": "Points to the PHP executable.",
```

Comment:
I'd call it
`php.suggest.builtIn': true
that' consistent to what we have in html

```
  // Configures if the built-in HTML language support suggests Ionic tags, properties and values.
  "html.suggest.ionic": true,
```